content/community/changelog/app-nuget/v4/whats-new/_index.en.md

```diff
@@ -4,6 +4,13 @@ description: Overview of changes introduced in version 4.
 toc: true
 ---
 
+## 4.20.0 (2021-11-15) - Added support for custom redirect URL when exiting an app
```

Comment:
Date here should be updated once we have the publish date for the final nuget package :)

Reply:
Date is 2021-11-18 ;)

Reply:
Superb!